chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
+
+const (
+	RawTx HexFormatFlag = 1
```

Comment:
Minor: lint error:
`exported: exported const RawTx should have comment (or a comment on this block) or be unexported (revive)go-golangci-lint`

Reply:
I've just put an output of linter. Consider add `"a comment on this block"` or just skit it and resolve this comment 😄